### ISSUE TITLE
Failed to open zip 9

### ISSUE THREAD

**Issue by ellisgl** [minecraft-linux/mcpelauncher-ui-manifest#66]
**Describe the bug**
Updating to 1.21.2.02 results in the following error:
Download failed
Error while extracting the downloaded file(s), please report this error:
Failed to open zip 9

**To Reproduce**
1. Go to 'Home tab'
2. Click on 'Download and Play'

**Expected behavior**
Should download the update and be able to play it.

**Screenshots**
![image](https://github.com/user-attachments/assets/e28fdb89-cc4d-4691-a17c-7003e6fa7e31)

**Desktop (please complete the following information):**
 - OS: Kubuntu 24.04 - KDE Plasma Version 5.27.11 - KDE Framework 5.115.0 - Qt Version - 5.15.13 - Kernel Version 6.8.0.38-generic 64-bit - Graphics Platform X11
 - Version: v0.15.2 (build 80)
 - Installation Type: Flatpak

**Additional context**
Where does it keep the temp downloads?

**Comment by ellisgl** [comment]
Looks like the flatpak version might be out of date? AppImage version works fine.

**Comment by ChristopherHX** [comment]
Doesn't happen for me on flatpak even after doing a full update, so this cannot be a library issue.

**I did instead get Failed to Open Output file errors, because my disk were full even outside of the flatpak sandbox**, removing an older version let it succeed without issues

Failed to open zip is usually an indicator that the download failed/corruped, but it the app didn't notice it.

I saw this message also once in our discord server, pretty recently.

Like you already did, as always use flatpak if the AppImage doesn't work and use the AppImage if the flatpak doesn't work.

Both have compatibility bugs and partially different glitches caused by the external Linux System.
